### PR TITLE
ci(lm-logs): change user-agent

### PR DESCRIPTION
**Pull request by siddharthck**
change user-agent to lm-logs-fluentd-k8s.
Upgrade base image to 1.2.0.

### CODE REVIEW

File: charts/lm-logs/templates/configmap.yaml
Comment (vkumbhar94) [inline charts/lm-logs/templates/configmap.yaml:60]: IMO, this value is kind of static - so it would be difficult to know the source cluster.
Similar to argus, take a `cluster name` which user can provide to uniquely identify their cluster.
I would recommend to take cluster name input if we don't already.
Format value like : `ll:fluentd:k8s:<clustername>`
CC: @shubhashri12

**Comment by shubhashri12** [comment]
Looks good to me.. Unfortunately i am not in the CODE-OWNERS list for this chart. Please get the approval from @vkumbhar94

**Review by vkumbhar94 — APPROVED** [review]

**Comment by shubhashri12** [comment]
Merging on behalf of @siddharthck

**Pull request merged**
